.github/workflows/readme-check.yml

```
@@ -0,0 +1,44 @@
+name: Check README Updates
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  check-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Find README.docify.md files and check generated READMEs
+        run: |
+          # Find all README.docify.md files
+          DOCIFY_FILES=$(find . -name "README.docify.md")
+
+          for file in $DOCIFY_FILES; do
+            echo "Processing $file"
+            
+            # Get the directory containing the docify file
+            DIR=$(dirname "$file")
+            
+            # Go to the directory and run cargo build
+            cd "$DIR"
+            cargo build --features generate-readme
```

How about using `check` instead of `build` and in case of failure print a message hinting to the `generate-readme` convention (besides the case of compilation errors):
```suggestion
            cargo check --features generate-readme || echo "Readme generation for $DIR failed. Ensure the crate compiles successfully and has a `generate-readme` feature which guards markdown compilation in the crate as follows: https://docs.rs/docify/latest/docify/macro.compile_markdown.html#conventions."
```